roles/mailserver/files/docker-compose.yml

```diff
@@ -126,8 +126,37 @@ services:
       - OPENCART_HOST=store.{{ domain_name }}
       - OPENCART_USERNAME=opencart
       - OPENCART_PASSWORD=changeThis
-      - SMTP_HOST=mailserver
-      - SMTP_PORT=25
+      - SMTP_HOST=mailserver 
+      - SMTP_PORT=25 
+      - SMTP_USER=username@domain.ext
+      - SMTP_PASSWORD=password
+{% endif %}
+
+{% if use_poweradmin %}
+  poweradmin:
+    image: secns/pdns
+    container_name: poweradmin
+    restart: on-failure
+    depends_on:
+      - mariadb
+    restart: on-failure
+    volumes:
+      - /mnt/docker/poweradmin:/poweradmin
+    ports:
+    - "53:53"
+    - "53:53/udp"
+    environment:
+    - PDNS_ALLOW_AXFR_IPS=127.0.0.1
+    - PDNS_DISTRIBUTOR_THREADS=3
+    - PDNS_CACHE_TTL=20
+    - PDNS_RECURSIVE_CACHE_TTL=10
+    - DB_ENV_MYSQL_ROOT_PASSWORD=changeme
+    - MARIADB_PASSWORD={{ db_password }}
+    - MYSQL_ROOT_PASSWORD=changeme
```

Comment:
What is `MYSQL_ROOT_PASSWORD` here? Is it for the `pdns` specific DB?

Might want ask the questions and add the `PDNS` related SQL password to the `secrets.yml`. See how variables like `db_password` are handled, instead of using `changeme`. I set a bad example. :-)

Reply:
the container uses a mysqldb ... i tried to figure out how to simply use the mariadb but it failed so i used the "docker-compose.yml" parameters from https://github.com/obi12341/docker-pdns.git

Reply:
Looking at that setup, this is what you need to do:

```
- DB_ENV_MYSQL_ROOT_PASSWORD={{ db_password }}
```
(remove both the MARIADB_PASSWORD and MYSQL_ROOT_PASSWORD stuff)

and add a `links` section:
```
links:
  - mariadb:db
```